recipes/pspincalc/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "PSpincalc" %}
+{% set version = "0.2.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1116da1f53762a8daff41361066c55d635f94de2b58b45e90eecdacf2112cbe2
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - numpy
```

Comment:
Do you need `numpy` here? Unless it's imported by the `setup.py` script you probably don't
```suggestion
```